cmd/sfncli/test_scripts/stdout_empty_output.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo ""
```

Comment:
in reality the workers we're talking about don't even echo a newline, so I'd consider removing this

Reply:
Good point.  Updated in c3a79bf1920c29877d768ec2233e14445d55aaa0